stream-monitor/monitor.liq

```diff
@@ -6,7 +6,7 @@ def blank_detected() =
     _ = get_process_lines("/data/silence.sh")
 end
 
-source = mksafe(input.https("https://stream.wuvt.vt.edu/wuvt.ogg"))
+source = mksafe(input.https(max=20., "https://stream.wuvt.vt.edu/wuvt.ogg"))
```

Comment:
probably should be `20`, not `20.`... please check liquidsoap documentation for this interface